src/main/resources/changelogs/db.changelog-add-river-name-to-fluvial-forecast-location-table.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet author="takanocap" id="changelog-add-rivername-to-fluvial-forecast-location-table">
+        <sql>
+            IF COL_LENGTH('FFF_STAGING.FLUVIAL_FORECAST_LOCATION', 'RIVER_NAME') IS NULL
+            ALTER TABLE FFF_STAGING.FLUVIAL_FORECAST_LOCATION ADD RIVER_NAME NVARCHAR(255) NULL
+        </sql>
+        <rollback>
+            <sql>
+                IF COL_LENGTH('FFF_STAGING.FLUVIAL_FORECAST_LOCATION', 'RIVER_NAME') IS NULL
+                ALTER TABLE FFF_STAGING.FLUVIAL_FORECAST_LOCATION DROP COLUMN RIVER_NAME

```

Comment:
The rollback script does not work.

-  The guard condition should be NOT NULL based.
-  RIVER_NAME in the ALTER TABLE statement should not be quoted.

Please replace with:
                IF COL_LENGTH('FFF_STAGING.FLUVIAL_FORECAST_LOCATION', 'RIVER_NAME') IS NOT NULL
                ALTER TABLE FFF_STAGING.FLUVIAL_FORECAST_LOCATION DROP COLUMN RIVER_NAME